pom.xml

```diff
@@ -35,11 +35,17 @@
       <artifactId>gson</artifactId>
       <version>2.8.6</version>
     </dependency>
+
+    <dependency>
+      <groupId>mysql</groupId>
+      <artifactId>mysql-connector-java</artifactId>
+      <version>8.0.19</version>
+    </dependency>
   </dependencies>
 
   <build>
     <plugins>
-      <!-- Provides `mvn package appengine:run` for local testing
+      <!-- Provides `mvn package appengine:run` for testing on the cloud
```

Comment:
You changed the comment but not the plugin? Is the original comment wrong or why the change?

Reply:
yeah I didn't change the plugin, it's just to differentiate that one is the dev server on appengine and the other is the local server on our laptops